cli/js/workers_test.ts

```diff
@@ -47,13 +50,17 @@ test(async function nestedWorker(): Promise<void> {
 
   nestedWorker.onmessage = (e): void => {
     assert(e.data.type !== "error");
-    promise.resolve();
+    // Defer promise.resolve() to allow worker to shut down
+    setTimeout(() => {
+      promise.resolve();
+    });
   };
 
   nestedWorker.postMessage("Hello World");
   await promise;
 });
 
+/* FIXME(bartlomieju): leaks async ops - add worker.terminate() to fix that
```

Comment:
Note to self: this worker is actually useless when it throws when executing module - there should be no pending op in that case.

Reply:
This comes from the fact that we never terminate `worker.poll()` loop - te be figured out